spot_driver/launch/spot_driver.launch.py

```diff
@@ -3,10 +3,11 @@
 import logging
```

Comment:
nit: we should update the copyright to the proper format: https://www.notion.so/theaiinstitute/Copyrights-a3c268b0583d4518826b9de690602d6c